src/terminologies/mdm-term.resource.ts

```diff
@@ -252,4 +255,24 @@ export class MdmTermResource extends MdmResource {
     const url = `${this.apiEndpoint}/terminologies/${terminologyId}/terms/${termId}/codeSets`;
     return this.simpleGet(url, queryStringParams, restHandlerOptions);
   }
+
+  /*
+    * `HTTP POST` - Copy a term.
+    *
+    * @param terminologyId The unique identifier of the terminology to copy the term to.
+    * @param termId The unique identifier of the term to copy.
+    * @param copyTermPayload The payload of the request containing all the details for the term to copy.
+    * @returns The result of the `POST` request.
+    *
+    * `200 OK` - will return a {@link TermDetailResponse} containing a {@link TermDetail} object.
+  */
+  copy(
+    terminologyId: Uuid,
```

Comment:
More consistency things:

- [ ] Move `target...` and `source...` properties into function parameters
- [ ] Make it clear which IDs are the source and which are the target

Reply:
targetTerminologyId is a payload requirement